clients/cli/src/main/java/org/apache/gravitino/cli/commands/DeleteGroup.java

```diff
@@ -28,29 +29,36 @@ public class DeleteGroup extends Command {
 
   protected final String metalake;
   protected final String group;
+  protected final boolean force;
 
   /**
    * Delete a group.
    *
    * @param url The URL of the Gravitino server.
    * @param ignoreVersions If true don't check the client/server versions match.
+   * @param force Force operation.
    * @param metalake The name of the metalake.
    * @param group The name of the group.
    */
-  public DeleteGroup(String url, boolean ignoreVersions, String metalake, String group) {
+  public DeleteGroup(
+      String url, boolean ignoreVersions, boolean force, String metalake, String group) {
     super(url, ignoreVersions);
+    this.force = force;
     this.metalake = metalake;
     this.group = group;
   }
 
   /** Delete a group. */
-  @Override
   public void handle() {
     boolean deleted = false;
 
+    if (!AreYouSure.really(force)) {
+      return;
+    }
+
     try {
       GravitinoClient client = buildClient(metalake);
-      deleted = client.removeGroup(group);
+      deleted = client.removeUser(group);
```

Comment:
why change the method from "removeGroup" to "removeUser" in the "DeleteGroup" action?

Reply:
Sorry that was a merge issue - fixed.